Firestore/Source/Local/FSTLevelDBRemoteDocumentCache.mm

```diff
@@ -84,6 +84,26 @@ - (nullable FSTMaybeDocument *)entryForKey:(const DocumentKey &)documentKey {
   }
 }
 
+- (MaybeDocumentMap)entriesForKeys:(const DocumentKeySet &)keys {
+  MaybeDocumentMap results;
+
+  LevelDbRemoteDocumentKey currentKey;
+  auto it = _db.currentTransaction->NewIterator();
+
+  for (const DocumentKey &key : keys) {
+    if (key > currentKey.document_key()) {
```

Comment:
This assumes DocumentKeySet is an ordered (rather than unordered) set. Which it is.

If you wanted to be super pedantic about it, you could rename to DocumentKeySortedSet (or ordered set).

Optional.